lib/services/matchers.go

```diff
@@ -39,23 +39,26 @@ type AWSMatcher struct {
 	Tags types.Labels
 }
 
-// MatchResourceLabels returns true if any of the provided selectors matches the provided database.
+// MatchResourceLabels returns true if all of provided ResourceMatchers matches database resource.
```

Comment:
Right now the MatchResourceLabels is used only in [DB](https://github.com/gravitational/teleport-private/blob/e3a8fb7a0fa7e32504a714c370c3a8dbf71b1d6f/lib/srv/db/watcher.go#L177 ) [APP](https://github.com/gravitational/teleport-private/blob/e3a8fb7a0fa7e32504a714c370c3a8dbf71b1d6f/lib/srv/app/watcher.go#L181) watchers